sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormTrainingClient/FormTrainingClientTests.cs

```diff
@@ -129,5 +162,13 @@ public void CreateFormRecognizerClientFromFormTrainingClient()
             Assert.IsNotNull(formRecognizerClient.Diagnostics);
             Assert.IsNotNull(formRecognizerClient.ServiceClient);
         }
+
+        private static string GetString(RequestContent content)
+        {
+            using var stream = new MemoryStream();
+            content.WriteTo(stream, CancellationToken.None);
+
+            return Encoding.UTF8.GetString(stream.ToArray());
+        }
```

Comment:
Method is repeated in `FormTrainingClientTests` and `FormRecognizerClientTests`. I don't think it would be worth to create a new common file for a single stream-to-string method with 3 lines of code. We should do it if more common methods are needed.